cider/function/extensions/substrait/substrait_extension.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML 1.2
+---
+scalar_functions:
+  -      
+    name: "between__3"
```

Comment:
Use Between__3 is because of Between__3 is supported by cider, which is register in ConverterHelper.cpp. 
Extention yaml file will been updated constantly.